src/export/StructureDefinitionExporter.ts

```diff
@@ -310,7 +310,8 @@ export class StructureDefinitionExporter implements Fishable {
     } else {
       delete structDef.title;
     }
-    structDef.status = 'active'; // it's 1..1 so we have to set it to something; can be overridden w/ rule
+    structDef.status = this.pkg.config.status;
```

Comment:
As noted in a previous comment: Unless there is a reason to use `this.pkg.config`, we should use `this.tank.config` since that's where we pull the config from when setting other config-based values in this same function.

Reply:
Addressed In commit [25e2afa](https://github.com/FHIR/sushi/pull/1143/commits/25e2afad1a873bf611c84ecca118ffc500f0d74e).